java-frontend/src/main/java/org/sonar/java/se/SymbolicExecutionVisitor.java

```diff
@@ -106,6 +115,32 @@ private boolean isObjectsRequireNonNullMethod(Symbol symbol) {
       return symbol.owner().type().is("java.util.Objects") && "requireNonNull".equals(symbol.name());
     }
 
+    @CheckForNull
+    private MethodBehavior createStringUtilMethodBehavior(Symbol.MethodSymbol symbol) {
+      MethodBehavior behavior;
+      switch (symbol.name()) {
+        case "isEmpty" :
+        case "isBlank" :
+        case "isNotEmpty" :
+        case "isNotBlank" :
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 12 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1151)